CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [4.2.0] - TBD
```

Comment:
Generally I don't bother pre-populating the changelog as it's not possible to tell what version number is incremented, but this works, we can always adjust :). I'm also unsure if perf improvements count as a feature or not under semver, I could see it going either way.

Reply:
I just checked https://semver.org/ again. I'm actually fixing a perf bug. Let's just increment the last digit then. I also made a commit to release 4.1.1 today.